content/1.7/back-office/login.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Login into Back-Office
+weight: 1
+---
+# LOGIN
+
+![Login UI](static/login.png)
```

Comment:
It is not recommended to use abbreviation in a spec without definition. Could you please either define UI (user interface) or change the word?

Reply:
@marwachelly, thanks for info, I have updated to "PRESTASHOP AUTHENTICATION WINDOW". Makes sense? :)